armada/tests/unit/handlers/test_wait.py

```diff
@@ -0,0 +1,29 @@
+from armada.handlers.armada import Armada
```

Comment:
I like that you have started adding testing for this feat. We need to increase our coverage its a big need. We should add more testing.